rkt/status.go

```diff
@@ -30,7 +30,7 @@ import (
 
 var (
 	cmdStatus = &cobra.Command{
-		Use:   "status [--wait=bool|timeout] [--wait-ready=bool|timeout] UUID",
```

Comment:
Thanks for fixing the trailing space! We try to keep commits to what they describe they're doing.

Can you please just remove it from the first commit? Right now it's added on the first commit just to be removed on the second one, that shouldn't be necessary if you don't add it in the first place.